apps/kahoot-it-clone/src/app/header/header.component.css

```diff
@@ -0,0 +1,17 @@
+.toolbar {
+  display: grid;
+  grid-template-areas: 'a b c d e';
```

Comment:
Make these meaningful. 'a b c d e' doesn't tell me anything about the structure. Rememeber everything needs to have meaning. I would rather you make a super long function name like thisAddsARecipeToTheArray() than something like add(). Although the former is exaggerated, pedantic and can be simplified and written better, it tells me what it is doing before I even start getting into its inner workings. Even if it seems simple to you, as the codebase grows and you forget about this later, when you come back to it you will appreciate it because you will have a better understanding on what it is doing.